fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -36,6 +36,28 @@ public static FHIRRegistry getInstance() {
         return INSTANCE;
     }
     
+    public boolean hasResource(String url) {
+        if (Objects.isNull(url)) {
+            return false;
+        }
+        
+//      String id = null;
+        int index = url.indexOf("#");
+        if (index != -1) {
+//          id = url.substring(index + 1);
```

Comment:
why is this commented out?  is it needed?

Reply:
removed